src/App.scss

```
@@ -1 +1,45 @@
-// style go here
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

do not set margin, padding and font-family to all elements. the font-family can be set for `html`
```suggestion
  box-sizing: border-box;
}

html {
  font-family: Verdana, Geneva, Tahoma, sans-serif;
```